x-pack/test/alerting_api_integration/spaces_only/tests/alerting/flapping_history.ts

```diff
@@ -0,0 +1,277 @@
+/*
```

Comment:
how painfully slow are these to run? :-)

We had a couple thoughts on trying to get them to run quicker:

- try changing the task manager polling interval to 1s - that might well break OTHER tests expecting the 3s interval, but ... worth a quick experiment.  We'd add that as a new option here: https://github.com/elastic/kibana/blob/main/x-pack/test/alerting_api_integration/spaces_only/config.ts

- try to get all the various tests to run in the same range of 20 or more rule executions, by using different pattern instances in the pattern rule - one would test going active on run 1, then not active for 20 runs, and never be flapping.  Another would become flapping ASAP, then be active for the next 20 runs, then no longer be flapping and fire the recovered actions.  Etc.